### ISSUE TITLE
Add new status ticket's on future versions

### ISSUE THREAD

**Issue by mikhey54** [osTicket/osTicket-1.7#43]
Now -> Open and Closed

Features -> Open, Closed, Answered, .....

Answered -> avto-closed (Closed) on 3 day and block Post Reply Client form

**Comment by protich** [comment]
I disagree that "Answered" should be a status - it's a state of a ticket (status vs. state). I like the idea of auto-closing answered tickets after X days.

That said, let's restrict github issues to bugs & code related issues. The feature requests should go to [osTicket forum](http://osticket.com/forums/), in my opinion.
